docs/pipelines/tasks/utility/powershell.md

```diff
@@ -129,16 +129,39 @@ Write-Host  "##vso[task.LogIssue type=error;]This is the error"
 ```
 
 > [!TIP]
-> 
+>
 > If you want this error to fail the build, then add this line:
 >  ```ps
 > exit 1
-> ``` 
+> ```
 
 ### ApplyVersionToAssemblies.ps1
 
 [Use a script to customize your build pipeline](../../scripts/powershell.md)
 
+### Call powershell script with multiple arguments
```

Comment:
Please spell it "PowerShell" then I'll merge this. Thanks!

Reply:
Done :)